apps/cookbook/src/app/examples/alert-example/alert-example.component.ts

```diff
@@ -37,22 +37,24 @@ this.modalController.showAlert(config);`;
   private alertClose$: Subject<void> = new Subject<void>();
 
   static readonly alertConfigWithDynamicValues = `const title$ = of('Need more time?');
-  const message$ = remainingSeconds$.pipe(
-    map((remainingSeconds) => \`Time remaining: \${remainingSeconds}\`)
-  );
-
-  const config: AlertConfig = {
-    title: title$,
-    icon: {
-      name: 'clock',
-      themeColor: 'warning',
-    },
-    message: message$,
-    okBtn: 'Logout',
-    cancelBtn: 'Take me back',
-  };
+const message$ = remainingSeconds$.pipe(
+  map((remainingSeconds) => \`Time remaining: \${remainingSeconds}\`)
+);
+const logoutText$ = translationService.translate('logout');
+const takeMeBackText$ = translationService.translate('take_me_back');
```

Comment:
Consider empasizing the made up part a bit
```suggestion
const logoutText$ = this.myTranslationService.get('logout');
const takeMeBackText$ = this.myTranslationService.get('take_me_back');
```